src/components/Board.js

```diff
@@ -19,67 +19,234 @@ const Board = ({ socket }) =>  {
     };
   }, [socket]);
 
-
   //Variables and constants
   //let mark = ''; // not yet used, removed so eslint would stop complaining
-  //let winner = ''; 
+  let winner = '';
   const reference = useRef(null);
 
   //Functions
   const move = (event, index) => {
     const newBoard = [...boardState];
     const curr = turn === 0 ? 'X' : 'O';
-    if(newBoard[index] === '') {
+    if (newBoard[index] === '') {
       newBoard[index] = curr;
       event.target.innerHTML = curr;
     }
-    
+
     setBoard(newBoard);
-    console.log(boardState);
-    socket.emit("move", { boardState: boardState });
-    setTurn(turn === 0 ? 1 : 0);
-  }
+    console.log(newBoard);
+    // check for winner
+    if (index === 0) {
```

Comment:
there might be a shorter way to check this with relative positions without having to do an if else for every single possible position?

Reply:
> there might be a shorter way to check this with relative positions without having to do an if else for every single possible position?

I think so - I'm thinking of leaving it as is and going back to it after working on other stuff. We each need to write 1000+ lines so I wasn't too concerned with longer codes, unless you think it's making the app run slower.

Reply:
honestly line of code metric is a bad way for her to grade this project. more elegant shorter code is always better. performance wise i don't think it matters though in this case. also, if you are worried about lines of code we can make the project a bit harder and do checkers instead of tic tac toe but yeah i see where you are coming from

Reply:
yeah...I figured we can add another small game or something if our tic-tac-toe ended up less than 3000 lines